ui/static/js/fortio_chart.js

```diff
@@ -372,7 +373,7 @@ function deleteSingleChart () {
   chart = {}
 }
 
-function makeMultiChart (data) {
```

Comment:
I know data is global but isn't it cleaner to pass it anyway ?

Reply:
`data` is not used in `makeMultiChart`. The data for the chart is added over time with `fortioAddToMultiResult`

Reply:
oh ! :( my bad ! ok some day we should objectify this whole thing but separately from adding features